config/sota_hsm_prov.toml

```diff
@@ -18,5 +19,3 @@ type = "sqlite"
 [import]
 base_path = "/var/sota/import"
 tls_cacert_path = "root.crt"
-tls_clientcert_path = "client.pem"
-tls_pkey_path = "pkey.pem"
```

Comment:
You're removing these because aktualizr won't need them, but rather the testtoken script, right?

Reply:
Yes, absolutely. `[import]` section is for importing stuff into storage, and our certificate/private key are in HSM, not in storage.